rpmlb/cli.py

```diff
@@ -73,6 +73,22 @@
     '--copr-repo', '-C', metavar='COPR_REPO',
     help='Target Copr for copr builder.',
 )
+@click.option(
+    '--epel', metavar='VERSION', type=int, default=7,
```

Comment:
Sorry my suggesting name `--koji-epel` was not good. 
How about a name `--rhel`?
Maybe it is used in RPM spec macro to describe what you want to use.

https://github.com/sclorg/rpm-list-builder/issues/48

```
%rhel 7
```